app/views/organisations/_show_organisation.html.erb

```diff
@@ -49,16 +49,34 @@
   <% end %>
 </div>
 
+<% if @organisation.ordered_featured_documents.length %>
```

Comment:
Should this check `latest` rather than ordered_featured_documents? We'll definitely need to make that change when we get the correct document data to show.

Reply:
Also I'm not sure this works. It would need to be something like:

`if @organisation.ordered_featured_documents.length > 0`
or
`if @organisation.ordered_featured_documents.any?`

At the moment, even if length is 0 the section will still display.

Reply:
Fixed.